src/StatusCodeInterface.php

```diff
@@ -37,71 +37,71 @@
-    const STATUS_SWITCHING_PROTOCOLS = 101;
-    const STATUS_PROCESSING = 102;
-    const STATUS_EARLY_HINTS = 103;
+    const 💯 = 100;
+    const 🔀 = 101;
+    const 🤔 = 102;
+    const 💭 = 103;
     // Successful 2xx
-    const STATUS_OK = 200;
-    const STATUS_CREATED = 201;
-    const STATUS_ACCEPTED = 202;
-    const STATUS_NON_AUTHORITATIVE_INFORMATION = 203;
-    const STATUS_NO_CONTENT = 204;
-    const STATUS_RESET_CONTENT = 205;
-    const STATUS_PARTIAL_CONTENT = 206;
-    const STATUS_MULTI_STATUS = 207;
-    const STATUS_ALREADY_REPORTED = 208;
-    const STATUS_IM_USED = 226;
+    const 👌 = 200;
+    const ✍️ = 201;
+    const ✔️ = 202;
+    const 📝 = 203;
+    const 🗅 = 204;
+    const 👍 = 205;
+    const 😩 = 206;
+    const 🤯 = 207;
+    const 📰 = 208;
+    const 💬 = 226;
     // Redirection 3xx
-    const STATUS_MULTIPLE_CHOICES = 300;
-    const STATUS_MOVED_PERMANENTLY = 301;
-    const STATUS_FOUND = 302;
-    const STATUS_SEE_OTHER = 303;
-    const STATUS_NOT_MODIFIED = 304;
-    const STATUS_USE_PROXY = 305;
-    const STATUS_RESERVED = 306;
-    const STATUS_TEMPORARY_REDIRECT = 307;
-    const STATUS_PERMANENT_REDIRECT = 308;
+    const 🔁 = 300;
+    const 🔂 = 301;
+    const 🔎 = 302;
+    const 😕 = 303;
+    const 😟 = 304;
+    const 😨 = 305;
+    const 🏚️ = 306;
+    const 🙃 = 307;
+    const 😬 = 308;
     // Client Errors 4xx
-    const STATUS_BAD_REQUEST = 400;
-    const STATUS_UNAUTHORIZED = 401;
-    const STATUS_PAYMENT_REQUIRED = 402;
-    const STATUS_FORBIDDEN = 403;
-    const STATUS_NOT_FOUND = 404;
-    const STATUS_METHOD_NOT_ALLOWED = 405;
-    const STATUS_NOT_ACCEPTABLE = 406;
-    const STATUS_PROXY_AUTHENTICATION_REQUIRED = 407;
-    const STATUS_REQUEST_TIMEOUT = 408;
-    const STATUS_CONFLICT = 409;
-    const STATUS_GONE = 410;
-    const STATUS_LENGTH_REQUIRED = 411;
-    const STATUS_PRECONDITION_FAILED = 412;
-    const STATUS_PAYLOAD_TOO_LARGE = 413;
-    const STATUS_URI_TOO_LONG = 414;
-    const STATUS_UNSUPPORTED_MEDIA_TYPE = 415;
-    const STATUS_RANGE_NOT_SATISFIABLE = 416;
-    const STATUS_EXPECTATION_FAILED = 417;
-    const STATUS_IM_A_TEAPOT = 418;
-    const STATUS_MISDIRECTED_REQUEST = 421;
-    const STATUS_UNPROCESSABLE_ENTITY = 422;
-    const STATUS_LOCKED = 423;
-    const STATUS_FAILED_DEPENDENCY = 424;
-    const STATUS_TOO_EARLY = 425;
-    const STATUS_UPGRADE_REQUIRED = 426;
-    const STATUS_PRECONDITION_REQUIRED = 428;
-    const STATUS_TOO_MANY_REQUESTS = 429;
-    const STATUS_REQUEST_HEADER_FIELDS_TOO_LARGE = 431;
-    const STATUS_UNAVAILABLE_FOR_LEGAL_REASONS = 451;
+    const 👎 = 400;
+    const 🔐 = 401;
+    const 💳 = 402;
+    const 🛑 = 403;
+    const 🚧 = 404;
+    const 🙅 = 405;
+    const 🚫 = 406;
+    const 🛂 = 407;
+    const ⌛ = 408;
+    const 😠 = 409;
+    const 👋 = 410;
+    const 📏 = 411;
+    const ❌ = 412;
+    const 💣 = 413;
+    const 🤐 = 414;
+    const 💔 = 415;
+    const 🍽️ = 416;
+    const 😐 = 417;
+    const 🤓 = 418;
+    const ✨ = 421;
```

Comment:
Where is `420`?

Reply:
I couldn't think of a comparable emoji for that but I'm open to suggestion

Reply:
![image](https://user-images.githubusercontent.com/16620420/55292241-451a1000-53b6-11e9-9771-af0d08324c89.png)

Reply:
I think 😤 would be appropriate.

Reply:
☕ would be more appropriate for 418 IMO.

Reply:
🍁